website/developer-docs/api/flow-executor.md

```diff
@@ -4,7 +4,7 @@ title: Flow executor (backend)
 
 A big focus of authentik is the flows system, which allows you to combine and build complex conditional processes using stages and policies. Normally, these flows are automatically executed in the browser using authentik's [standard browser-based flow executor (/if/flows)](/docs/flow/executors/if-flow).
 
-However, any flow can be executed via an API from anywhere, in fact that is what every flow executor does. With a few requests you can execute flows from anywhere, and integrate authentik even better.
+However, any flow can be executed via an API from anywhere, in fact that is what the backend flow executor does. With a few requests you can execute flows from anywhere, and integrate authentik even better.
```

Comment:
This doesn't make sense, the backend provides the API, in this sense `every` meaning the default executor, sfe, user settings and ldap/radius one